src/xdocs/checks/coding/illegaltoken.xml

```diff
@@ -34,12 +34,10 @@
             <tr>
               <td>tokens</td>
               <td>tokens to check</td>
-              <td>set of any supported
-                <a href="../../apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html">
-                    tokens</a>
-              </td>
+              <td>TokenTypes</td>
               <td>
```

Comment:
Can we avoid this change ?
Link was inside TD tag, not now it is out, and text is changed

Reply:
@romani, 

```
              <td>TokenTypes
                 <a href="../../apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#LABELED_STAT">
                    LABELED_STAT</a>
                  .
              </td>
```
              
              Ideally, if it looks this way, will it be acceptable? Are you comfortable with `set of any supported tokens` being replaced `TokenTypes`?

Reply:
>Are you comfortable with set of any supported tokens being replaced TokenTypes?

If we can avoid this update it will be good, as change of this will evolve bunch of maintainers to agree on it.

If this is not easy, ok, but get agreement with all will also be not easy.

Reply:
i assume i have tried to restore the original. @romani, Please let me know your feedback. Initially, it seemed it was related to https://github.com/checkstyle/checkstyle/issues/13885. But, if these changes gets approved, then the https://github.com/checkstyle/checkstyle/issues/13885 won't be requiring any changes.